roles/servicetelemetry/tasks/component_prometheus_reader.yml

```diff
@@ -0,0 +1,54 @@
+- name: Create ServiceAccount/stf-prometheus-reader
+  k8s:
+    definition:
```

Comment:
You could add the following line to these `k8s` module calls to allow STO to also remove the extra ServiceAccount, Role, and RoleBinding if the metrics storage is ever disabled (could happen in the future?):

```yaml
state: '{{ "present" if servicetelemetry_vars.backends.metrics.prometheus.enabled else "absent" }}'
```